index.html

```diff
@@ -16,9 +19,38 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="row">
+    <div class="container column-full">
+      <h2>Code Journal</h2>
+    </div>
   </header>
   <main>
+    <div class="container" data-view="entry-form">
+      <div class="entry-title column-full">
+        <h1>New Entry</h1>
+      </div>
+      <form id="journal-entry">
+        <div class="img-url-title column-full row">
+          <img class="column-half" src="./images/placeholder-image-square.jpg" alt="placeholder image" style="padding: 0" id="preview">
+          <div class="title column-half">
+            <p> Title</p>
+            <label for="Title"></label>
+            <input class="column-full text-padding" type="text" name="title" id="title" required>
+            <p>Photo URL</p>
+            <label for="PhotoURL"></label>
+            <input class="column-full text-padding" type="url" name="PhotoURL" id="photo" required>
+          </div>
+        </div>
+        <div class="notes row">
+          <p>Notes</p>
+          <textarea name="notes" class="column-full text-padding" rows="10"></textarea>
```

Comment:
you don't want the column class to be the content, you want to wrap the text area and label with a div with a class of column-full

remember columns are not content, they contain content.